src/criteria.rs

```diff
@@ -177,6 +177,16 @@ impl CriteriaMapper {
     pub fn criteria_index(&self, criteria_name: CriteriaStr<'_>) -> usize {
         self.index[criteria_name]
     }
+
+    /// Yields the indicies for all criteria which imply the the given criteria.
```

Comment:
another nit (suggestion aggregated with @anforowicz's comment)
```suggestion
    /// Yields the indices for all criteria which imply the given criteria.
```